governance/handler.go

```diff
@@ -65,6 +70,26 @@ var GovernanceItems = map[int]check{
 	params.Timeout:                 {uint64T, checkUint64andBool, nil},
 }
 
+func updateLowerBoundBaseFee(g *Governance, k string, v interface{}) {
+	g.ChainConfig.Governance.Reward.LowerBoundBaseFee = v.(uint64)
+}
+
+func updateUpperBoundBaseFee(g *Governance, k string, v interface{}) {
+	g.ChainConfig.Governance.Reward.UpperBoundBaseFee = v.(uint64)
+}
+
+func updateGasTarget(g *Governance, k string, v interface{}) {
+	g.ChainConfig.Governance.Reward.GasTarget = v.(uint64)
+}
+
+func updateBlockGasLimit(g *Governance, k string, v interface{}) {
+	g.ChainConfig.Governance.Reward.BlockGasLimit = v.(uint64)
+}
+
+func updateBaseFeeDenominator(g *Governance, k string, v interface{}) {
+	g.ChainConfig.Governance.Reward.BaseFeeDenominator = v.(uint64)
+}
+
```

Comment:
Please do not modify `Governance.ChainConfig`. It is intended to store the initial chain config.

What can change is BlockChain.chainConfig. If you want to edit BlockChain.chainConfig, then you should do similar to https://github.com/klaytn/klaytn/blob/v1.8.3/blockchain/blockchain.go#L429. and https://github.com/klaytn/klaytn/blob/v1.8.3/governance/handler.go#L80.

```go
// blockchain/blockchain.go
func (bc *BlockChain) GasTarget() uint64 { ... }
func (bc *BlockChain) SetGasTarget(val uint64) { ... }

// governance/handler.go
func updateBlockGasLimit(g *Governance, k string, v interface{}) {
	if g.blockChain != nil {
		g.blockChain.SetGasTarget(g.GetGovernanceValue(params.GasTarget).(uint64))
	}
}
```